eth/ethconfig/config.go

```diff
@@ -163,6 +163,8 @@ type Config struct {
 
 	// OverrideVerkle (TODO: remove after the fork)
 	OverrideVerkle *uint64 `toml:",omitempty"`
+
+	CollectWitnesses bool `toml:",omitempty"`
```

Comment:
Please don't add it here, this will forever make this temp flag art of our APIs.

Reply:
Also FYI, you should have regenerated the gen_config file when touching this, otherwise it's a noop.